app/export.go

```diff
@@ -269,11 +269,17 @@ func (a *App) buildUserTeamAndChannelMemberships(userId string) (*[]UserTeamImpo
 		memberData := ImportUserTeamDataFromTeamMember(member)
 
 		// Do the Channel Memberships.
-		channelMembers, err := a.buildUserChannelMemberships(userId, member.TeamId)
+		channelMembers, err := a.buildUserChannelMemberships(userID, member.TeamId)
 		if err != nil {
 			return nil, err
 		}
 
+		// Get the user theme
+		themePreference, err := a.Srv.Store.Preference().Get(member.UserId, model.PREFERENCE_CATEGORY_THEME, "")
```

Comment:
You are getting here the main theme preference, probably you want to pass the "teamId" in the last parameter to get the Theme preference for the team

Reply:
Ah didn't know about that. Thanks.